### DISCUSSION TITLE
Bun support

### DISCUSSION THREAD

**Discussion by igas** [resend/react-email#919] [Ideas]
### Goals

To be able to run react-email with bun

### Background

We recently moved all of our projects to bun, it would be good to support it here too.

### Proposal

Update `detect-package-manager` with renovate to add the support. You can trigger update from dashboard https://github.com/resendlabs/react-email/issues/746

**Comment by shellscape** [comment]
I'm working on a replacement for the preview and CLI which doesn't need to dynamically install dependencies. Will keep everyone posted.
